ext/semian/resource.c

```diff
@@ -53,7 +64,9 @@ semian_resource_acquire(int argc, VALUE *argv, VALUE self)
     if (res.error == EAGAIN) {
       rb_raise(eTimeout, "timed out waiting for resource '%s'", res.name);
     } else {
-      raise_semian_syscall_error("semop()", res.error);
+      char semerr[40];
+      sprintf(semerr, "semop() %d", res.sem_id);
```

Comment:
This is for debugging (saying what semaphore fucked up), will be removed before merging.